### ISSUE TITLE
wrong image src in templates

### ISSUE THREAD

**Issue by natalya-semenova** [gkushang/cucumber-html-reporter#96]
With new 'Save screenshots into custom directory' feature (https://github.com/gkushang/cucumber-html-reporter/pull/88) `step.image` that is used in templates as the URL of the image points to the wrong location (relative path is not correct)

This `step.image = filename;`  https://github.com/gkushang/cucumber-html-reporter/blob/30b2d658f8cd10156a76632cc8123a62e706be94/lib/reporter.js#L237

Example:
```javascript
const report = require('cucumber-html-reporter');
const { argv } = require('yargs');

const jsonFile = _.get(argv, 'src', 'test/e2e/report/cucumber_report.json');
const output = _.get(argv, 'dest', 'test/e2e/report/cucumber_report.html');
const storeScreenShots = _.get(argv, 'storeScreenShots', true);

report.generate({
  theme: 'bootstrap',
  jsonFile,
  output,
  storeScreenShots,
});
```

This will generate Cucumber HTML report test/e2e/report/cucumber_report.html 
with images source pointing to `"/test/e2e/report/screenshots/After_XX.png"` instead of `"screenshots/After_XX.png"`

**Comment by hayorov** [comment]
Plz, plz!

**Comment by gkushang** [comment]
This has been resolved and released with `cucumber-html-reporter@2.0.2`

**Comment by gkushang** [comment]
@natalya-semenova  

Please try the latest version & feel free to reopen the issue for any further feedback.

Thanks!
